wolfssl/ssl.h

```diff
@@ -3549,6 +3549,7 @@ WOLFSSL_API size_t wolfSSL_get_client_random(const WOLFSSL* ssl,
 
 #if defined(OPENSSL_EXTRA) || defined(OPENSSL_ALL)
 WOLFSSL_API size_t wolfSSL_BIO_wpending(const WOLFSSL_BIO *bio);
+WOLFSSL_API int wolfSSL_BIO_supports_pending(const WOLFSSL_BIO *bio);
```

Comment:
can this be moved to a static local function or just a local_function (not declared in ssl.h)

Reply:
This could be useful for users as well to determine if pending is supported by the current BIO. This could also be useful in other areas where we call pending.